src/lucky_flow.cr

```diff
@@ -16,8 +16,9 @@ class LuckyFlow
     setting base_uri : String
     setting retry_delay : Time::Span = 10.milliseconds
     setting stop_retrying_after : Time::Span = 1.second
-    setting chromedriver_path : String?
```

Comment:
Want to add a method called chromedriver_path that does a compile-time raise to say it was renamed?